File: Functions/Notes.py
# Evelyn Chennault, Function Notes Python

# Functions hold actions to be reused

# camelCase
# snake_case (mostly for python)

# number = int(input("Please tell me a number:\n"))
# def add(numOne, numTwo): # Parameters set the name of the variable (just for the function)
    # return numOne+numTwo

# print(add(number,21)) # Arguements set the value of the variable just for that instance of the function
# addition = add(6,4)
# add(8,12)
# add(64,)
# add(11,10000)
# add(87,3)

def values(type):
    return input(f"Please give me a {type}:\n")

name = values("name")
place = values("place")
verb = values("verb (past tense ONLY)")

print(f"{name} was really fast getting to {place} because they {verb} the whole way there.")

File: Functions/Notes.c
//Evelyn Chennault, Functions Notes C
#include <stdio.h>

//int num;
//char name[50], place[50], verb[50];

//void add(int numOne, int numTwo){
    //return numOne+numTwo;
//}

void due(char assignment[50], char day[20]){
    printf("The %s assignment is due %s.", assignment, day);
}


int main(void){
    //printf("Please tell me a number:\n");
    //scanf("%d", num);
    //add(num,10);
    //add(1,8);
    //printf (add(72,5));
    due("Functions Notes", "Today");
    due("Hello World Update", "Tomorrow");
    due("Functions Notes", "Friday");
    return 0;
}